src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -694,3 +695,79 @@ export function getEnvTrackingInfo( slug: string ): any {
 		};
 	}
 }
+
+export class DevEnvSQLImportCommand {
```

Comment:
Would you mind moving this class into a separate file?

Reply:
IDK, maybe create `commands` directory and put it there? Then we can refactor other commands as well, and we will have them in a single directory. What do you think?

Reply:
Sounds like a neat idea :+1:

Reply:
👋 I've been thinking about this and I feel like `commands` aren't a true representation of what this code is. This code is helper functions/code wrapped in a class. It is not a command. 

I'm okay (I think) with refactoring into a separate directory but I think we should come up with a better name that commands. People (like me) will confuse it with the actual commands that are in the `bin` folder. 

I would recommend moving these into a subfolder with the `lib` directory (that already contains helper functions/modules) and name it to represent exactly what it is `lib-classes` or `helper-classes`.